Export/Scripts/legionPassives.lua

```diff
@@ -0,0 +1,195 @@
+if not loadStatFile then
+	dofile("statdesc.lua")
+end
+--loadStatFile("stat_descriptions.txt")
+loadStatFile("passive_skill_stat_descriptions.txt")
+
+local out = io.open("../Data/3_0/LegionPassives.lua", "w")
+
+local stats = dat"Stats"
+local alternatePassiveSkillDat = dat"AlternatePassiveSkills"
+local alternatePassiveAdditionsDat = dat"AlternatePassiveAdditions"
+
+local LEGION_PASSIVE_GROUP = 1e9
+
+---@type fun(thing:string|table|number):string
+function stringify(thing)
+	if type(thing) == 'string' then
+		return thing
+	elseif type(thing) == 'table' then
+		local s = "{";
+
+		for k,v in pairs(thing) do
+			--thing[k] = stringify(v);
+			s = s.."\n\t"
+
+			if type(k) == 'number' then
+				s = s.."["..k.."] = "
+			else
+				s = s.."[\""..k.."\"] = "
+			end
+
+			if type(v) == 'string' then
+				s = s.."\""..stringify(v).."\", "
+			else
+				if type(v) == "boolean" then
+					v = v and "true" or "false"
+				end
+
+				val = stringify(v)..", "
+
+				if type(v) == "table" then
+					val = string.gsub(val, "\n", "\n\t")
+				end
+
+				s = s..val;
+			end
+		end
+
+		return s.."\n}"
+	elseif type(thing) == 'number' then
+		return ""..thing;
+	end
+end 
+
+---@type table <string, table> @this is the structure used to generate the final data file Data/3_0/LegionPassives
+local data = {};
+data.nodes = {};
+data.groups = {};
+data.additions = {};
+
+local ksCount = -1;
+
+for i=1, alternatePassiveSkillDat.rowCount do
```

Comment:
Please check https://pathofexile.gamepedia.com/Passive_Skill_Tree_JSON to see what the different attributes are and update your documentation accordingly.